internal/pkg/cli/job_init.go

```diff
@@ -222,7 +224,7 @@ func (o *initJobOpts) Execute() error {
 		}
 	}
 	// If the user passes in an image, their docker engine isn't necessarily running, and we can't do anything with the platform because we're not building the Docker image.
-	if o.image == "" {
+	if o.image == "" && !o.manifestExists {
```

Comment:
Can we also fix the first scenario here: https://github.com/aws/copilot-cli/issues/3639#issuecomment-1150406758 (where users enter empty string "" on Image prompt) and the CLI blows up. We should validate that empty string isn't valid 🥺

Reply:
I feel like --dockerfile xor --image should be required if there isn't a manifest 🤔 
They can still change it afterwards, but doing `copilot deploy` after a `svc init` should ideally just work. Was there a scenario that you had in mind?

Reply:
> doing copilot deploy after a svc init should ideally just work.

This makes sense to me

> Was there a scenario that you had in mind?

Perhaps if the user wants to build & use their own image and they haven't built the image they want to use yet 👀. I personally also feel more comfortable editing configurations in an editor, instead of in CLI prompts.

A clear pro of having them required though is like you said about the deployment flow, and it reenforces that `image` is a required manifest configuration.